systemtests/utils/utils.go

```diff
@@ -296,7 +295,20 @@ func GetIpAddress(t *testing.T, node TestbedNode, ep string) string {
 		t.Fatalf("Error '%s' getting ip for ep %s, Output: \n%s\n",
 			err, ep, output)
 	}
-	return string(output)
+
+	for _, str := range strings.Split(string(output), "\\n\\t") {
+		if strings.HasPrefix(str, "IpAddress") {
+			ret := strings.SplitN(str, ":", 2)
+			if len(ret) < 2 {
+				break
+			}
+
+			ret = strings.Split(ret[1], ",")
```

Comment:
Minor: Instead of split strings.TrimPrefix(contName, ",") would be more explicit